metadata.json

```diff
@@ -1,10 +1,10 @@
 {
     "uuid": "GoogleEarthWallpaper@neffo.github.com",
-    "shell-version": ["3.36", "3.38", "40", "41", "42", "43", "44"],
+    "shell-version": ["45", "46", "47", "48"],
     "name": "Google Earth Wallpaper",
     "settings-schema": "org.gnome.shell.extensions.googleearthwallpaper",
     "description": "Sets your wallpaper to a random photo from the curated Google Earth collection (2604 photos).\n\n*Disclaimer*: this extension is unofficial and not affiliated with Google in any way. Images are protected by copyright and are licensed only for use as wallpapers.\n\nSee also my other extension, Bing Wallpaper Changer (https://github.com/neffo/bing-wallpaper-gnome-extension).\n\nFeatures:\n* Fetches a random Google Earth wallpaper and sets as both lock screen and desktop wallpaper\n* User selectable refresh intervals (default is once per day)\n* View location on Google Maps, Bing Maps, Gnome Maps, OpenStreetMaps\n*  German, Dutch and Chinese translations\n\nPlease report any bugs or suggestions to extension GitHub page below.",
-    "version": "17",
+    "version": "19",
```

Comment:
After merging version-18 and actually changing the version number, this change will create a conflict

Reply:
I think gnome extensions overwrites this anyway, but we can fix before upload.